src/software/ai/evaluation/BUILD

```diff
@@ -42,8 +42,14 @@ cc_test(
 
 cc_library(
     name = "calc_best_shot",
-    srcs = ["calc_best_shot.cpp"],
-    hdrs = ["calc_best_shot.h"],
+    srcs = [
+        "calc_best_shot.cpp",
+        "calc_best_shot_impl.cpp",
+    ],
+    hdrs = [
+        "calc_best_shot.h",
+        "calc_best_shot_impl.h",
```

Comment:
calc_best_shot_impl.h should be a src since this header should be private

Reply:
Done.